pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -251,71 +254,120 @@ func StartLibvirt(stopChan chan struct{}) {
 	}()
 }
 
-func StartVirtlog(stopChan chan struct{}, domainName string) {
-	go func() {
-		for {
-			var args []string
-			args = append(args, "-f")
-			args = append(args, "/etc/libvirt/virtlogd.conf")
-			// #nosec No risk for attacket injection. Args  are predefined strings
-			cmd := exec.Command("/usr/sbin/virtlogd", args...)
+func startVirtlogdLogging(stopChan chan struct{}, domainName string) {
+	for {
+		// #nosec No risk for attacket injection. Args  are predefined strings
```

Comment:
I think you can now remove the "nosec" comment line. This is used to skip gosec validation , you no longer have now. (you now use constant strings and not variables in exec)